Tools/markings/markings/prototypes.go

```diff
@@ -68,6 +68,7 @@ type MarkingPrototype struct {
 	MarkingCategory    string            `yaml:"markingCategory"`
 	SpeciesRestriction []string          `yaml:"speciesRestriction,omitempty"`
 	Sprites            []SpriteSpecifier `yaml:"sprites"`
+    Shader             string?           `yaml:"shader"` // impstation
```

Comment:
```suggestion
	Shader             string?           `yaml:"shader"` // impstation
```
Space to tabs for consistency.